### PR TITLE
Added BCH underlyings

### PR DESCRIPTION
**Pull request by kyting**

### CODE REVIEW

File: share/underlyings.yml [hunk 1]
```diff
@@ -7098,6 +7098,198 @@ frxBTCSEK:
...
+  exchange_name: FOREX
+  instrument_type: forex
+  market: forex
+  market_convention:
+    atm_setting: atm_delta_neutral_straddle
+    bf: 2_vol
+    delta_premium_adjusted: 1
+    delta_style: spot_delta
+    rr: call-put
+  pip_size: 0.01
```
Comment (yngshan) [inline share/underlyings.yml]: Is this the correct pipsize?

**Comment by yngshan** [comment]
@kyting , I noticed the pip size of all the BCH pairs is 0.01, that is confirm rifght?

BTW, there are two BTCEUR in the existing file, do you mind to delete one of it? One is with 0.01 and one is with 0.0001

**Comment by kyting** [comment]
@yngshan sure, which one do I delete?

**Comment by yngshan** [comment]
We are getting from provider with 4 decimal, so delete the one with 0.01

**Comment by kyting** [comment]
Ok done, thanks!

**Pull request closed without merging**